### PR TITLE
Fix Single Term Page #20 #2

### PR DESCRIPTION
**Pull request by dr5hn**
@ruf-io 

This PR fixes:
- #20 Create Meta Tags for Individual Terms
- #2 Style Term Page
- Added description in each terms markdown file.

Screenshot 2020-10-04 at 10 54 49 PM

### CODE REVIEW

**Review by ruf-io — COMMENTED** [review]
@dr5hn I love the improved style on the single page, but I have a proposal for a simpler way to add descriptions:

Instead of requiring a new description front matter param, what if we used the hugo [.Summary](https://gohugo.io/content-management/summaries/) page variable to auto generate description from content?

I think we would also have to automatically shorten it to 150 chars or less, but I think that can be done with the substring function in hugo.

This would allow us to remove `description` front matter from all definitions, keeping them really simple.

What do you think?

**Comment by dr5hn** [comment]
@ruf-io 
- it can't be auto generated since it's not being store in variable, it is printed as it is.
- in order to extract substring we need to store the data in a variable, and then do the magic stuffs to create a substring and i don't think this is a right way to do it.

File: layouts/terms/single.html [hunk 1]
```diff
@@ -1,37 +1,78 @@
...
+    crossorigin="anonymous"></script>
+  <script src="/toggle.js"></script>
 
-    <main>
-        
+  <!-- Primary Meta Tags -->
+  <title>{{ .Site.Title }}</title>
+  <meta name="title" content="{{ .Site.Title }}">
+  <meta name="description"
+    content="{{if .Site.Params.description}}{{ .Site.Params.description }}{{else}}{{.Description}}{{end}}">
```
Comment (ruf-io) [inline layouts/terms/single.html]:
```suggestion
  <meta name="description"
    content="{{ substr .Summary 0 150 }}…">
```

@dr5hn Try this ☝️ - the `.Summary` variable is some hugo magic that takes the `.Content` and strips out formatting. So if we substring it to 150 chars and add an ellipsis I think we get a pretty good description without adding an extra required param to the spec.

What do you think?

**Review by ruf-io — CHANGES_REQUESTED** [review]
Take a look at my suggestion, I'm being a stickler here because I want to keep the definition spec as simple as possible. :)

File: layouts/terms/single.html (hunk 1 above)
Comment (dr5hn) [reply layouts/terms/single.html]: @ruf-io Great, Thanks for reference.

But i did it better way, I used
`{{ .Summary | truncate 150  }}`

truncate - Does truncating chars upto a limit and adds an ellipsis at the end by it self.

**Review by ruf-io — APPROVED** [review]
LGTM

**Pull request merged**